src/client/ui/selector-inspector-panel/element-picker.js

```diff
@@ -0,0 +1,97 @@
+import hammerhead from './../deps/hammerhead';
+import testCafeCore from './../deps/testcafe-core';
+
+import { getChildren } from './utils/ui-root';
+
+import { selectorGenerator } from './selector-generator';
+import { highlighter } from './highlighter';
+import { tooltip } from './tooltip';
+
+const listeners = hammerhead.eventSandbox.listeners;
+
+const styleUtils   = testCafeCore.styleUtils;
+const serviceUtils = testCafeCore.serviceUtils;
+
+export const ELEMENT_PICKED = 'element-piked';
+
+class ElementPicker extends serviceUtils.EventEmitter {
+    actualSelectors;
+    actualTarget;
+    hiddenTestCafeElements = new Map();
+    handlers;
+
+    constructor () {
+        super();
+
+        this.handlers = {
+            onClick:     this._getClickHandler(),
```

Comment:
Why do we need to call the getHandler function?
Why we just can't bind the `onclick` property to handler directly?